share/sv.po

```diff
@@ -0,0 +1,76 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-11-16 17:49+0100\n"
+"PO-Revision-Date: 2021-11-16 17:49+0100\n"
+"Last-Translator: mats.dufberg@iis.se\n"
+"Language-Team: Zonemaster project\n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "Domain name required"
+msgstr "Domännamn är obligatoriskt"
+
+msgid ""
+"The domain name is not a valid IDNA string and cannot be converted to an A-"
+"label"
+msgstr ""
+"Domännamnet är inte ett giltigt IDN-namn och kan inte konverteras till A-"
+"labelformat"
+
+msgid "The domain name contains non-ascii characters and IDNA is not installed"
+msgstr ""
+"Domännamnet innehåller icke-ASCII-tecken, men stöd för IDN är inte "
+"installerat"
+
+msgid "The domain name character(s) are not supported"
+msgstr "Domännamnets teckeninnehåll stöds inte"
```

Comment:
How about this?

> Domännamnet innehåller tecken som inte stöds

Reply:
That could work, but that is a freer translation.